swap_meet/item.py

```diff
@@ -1,2 +1,20 @@
 class Item:
-    pass
\ No newline at end of file
+
+    def __init__(self, age = 0, condition=0, category=""):
+        self.category = category
+        self.condition = condition
+        self.age = age
+
+    def __str__(self):
+        return "Hello World!"
+
+    def condition_description(self):
+        conditions = {
```

Comment:
We could pull this dictionary out into the main module and use upper-case letters to indicate it's a constant value. We could also have used a tuple. In either case, you might want to handle cases where the key (condition) isn't in the dictionary's keys or even when it's not number.